roles/openshift_node_certificates/meta/main.yml

```diff
@@ -13,4 +13,4 @@ galaxy_info:
   - cloud
   - system
 dependencies:
-- role: openshift_ca
+- role: openshift_facts
```

Comment:
Should we add some type of a guard here in the off chance that someone tries to apply this role before the CA has been created on the ca host?